cypress/support/step-definitions/common-steps.js

```diff
@@ -44,6 +46,10 @@ Given('I open in full screen Test {string} component page in noIframe', (compone
   visitComponentUrl(component, 'in_full_screen_dialog', true, 'test-');
 });
 
+Given('I open {string} component page full in iframe', (component) => {
```

Comment:
the same here about convention, if we have `Test` directory we need to build step like `I open Test {string} component ...`